marklogic-unit-test-modules/src/main/ml-modules/root/test/default.xqy

```diff
@@ -107,8 +107,8 @@ declare function local:main() {
 						<tr>
 							<th><input id="checkall" type="checkbox" checked="checked"/>Run</th>
 							<th>Test Suite</th>
-							<th>Total Test Count</th>
-							<th>Tests Run</th>
+							<th>Total Tests</th>
```

Comment:
I tried this out - let's do "Tests" and "Assertions" - since numbers are shown for all 4 columns, I think it's obvious that these are counts.